Source/WebKit/UIProcess/RemoteLayerTree/mac/RemoteLayerTreeDrawingAreaProxyMac.mm

```diff
@@ -118,7 +118,8 @@ explicit RemoteLayerTreeDisplayLinkClient(WebPageProxyIdentifier pageID)
 
 DisplayLink& RemoteLayerTreeDrawingAreaProxyMac::displayLink()
 {
-    ASSERT(m_displayID);
+    if (!m_displayID)
+        return;
```

Comment:
`displayLink()` is guarantee to return a l-value ref to the DisplayLink object. It might be the case where the caller should be calling `existingDisplayLink()` instead and handling the early return by itself. `existingDisplayLink()` can return `nullptr`.